pkg/loop/internal/ccip/test/onramp_test.go

```diff
@@ -65,12 +58,12 @@ func TestOnRampGRPC(t *testing.T) {
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
-		<-shutdown
+		tests.RequireSignal(t, shutdown, "Failed to receive shutdown signal")
```

Comment:
i really dislike this.

Reply:
This was actually why I wanted to open the PR - I was worried about shutdown hanging indefinitely and preventing the test from closing (and then saw the opportunity to reduce some redundancy in the setup used by all the tests). 

In my first iteration on this PR I implemented RequireSignal by hand (i.e. a for loop to check if the shutdown had received anything or if we had time'd out), and then later saw that Jordan had beat me to it a while back.